db/primary_migrate/20240718180303_add_selected_email_to_identity.rb

```diff
@@ -0,0 +1,5 @@
+class AddSelectedEmailToIdentity < ActiveRecord::Migration[7.1]
+  def change
+    add_column :identities, :email_address_id, :bigint
```

Comment:
let's define a `has_one :email_address` relation on the `ServiceProviderIdentity` model?

Reply:
Added has_one to that model.